single-cell-ebi-gxa.yaml

```diff
@@ -0,0 +1,196 @@
+---
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+  - name: dropletutils_read_10x
+    owner: ebi-gxa
+    tool_panel_section_label: 'Single Cell'
```

Comment:
Can we make this "Human Cell Atlas" or something different? We do have a category currently called single-cell and we can not filter based on tools-ids (just now). So we filter per sections atm.

Reply:
Would 'HCA-Single Cell' do?

Reply:
Think so yes.